erica_app/erica/routes.py

```diff
@@ -109,6 +109,14 @@ def revoke_unlock_code(unlock_code_revocation: UnlockCodeRevocationData, include
         raise HTTPException(status_code=422, detail=e.generate_error_response(include_elster_responses))
 
 
+@app.get(ERICA_VERSION_URL + '/tax_offices/', status_code=status.HTTP_200_OK)
+def get_tax_offices():
+    """
+    The list of tax offices for all states is requested and returned.
+    """
+    return FileResponse("erica/static/tax_offices.json")
```

Comment:
Okay, I feel like I missed something here: Why don't we request the tax_offices at Elster, but instead only have a file?

Reply:
The idea is basically to reduce latency